### PR TITLE
W-13763497 Review "local-api-specification-tutorial.adoc"

### PR DESCRIPTION
**Pull request by chris-campbell9**

### CODE REVIEW

File: modules/ROOT/pages/local-api-specification-tutorial.adoc [hunk 1]
```diff
@@ -16,13 +18,16 @@ See xref:setup.adoc[] for more information.
 
 To create a new API specification:
 
-. In your web IDE instance, navigate to your Anypoint Code Builder view:
-+
-image::anypoint-code-builder-view.png[]
-. Select *Design an API*.
-+
-image::design-api-1.png[]
+. Open the Command Palette (`ctrl/cmd + shift + P`) and type _MuleSoft: Design an API_.
```
Comment (dukesphere) [inline modules/ROOT/pages/local-api-specification-tutorial.adoc]:
```suggestion
. Open the Command Palette by pressing Ctrl+Shift+p (Mac) or Cmd+Shift+p (Windows),  and type _MuleSoft: Design an API_.
```

File: modules/ROOT/pages/local-api-specification-tutorial.adoc [hunk 2]
```diff
@@ -82,6 +87,8 @@ version: 1.0.development
 
 == Implement the Local API
 
+// TODO: cc: Should we define "Local API"? What does this mean in the context of the Web IDE? Also, will customers know what a "multi-root workspace is"? Should this be defined? Should it be "multi-version workspace"?
```
Comment (dukesphere) [inline modules/ROOT/pages/local-api-specification-tutorial.adoc:90]: Totally agree! For web, it's basically what's created in the IDE instance associated with your Anypoint Platform user name/pw account. No one else can access it but you bc it requires your credentials to open it. You'd have to publish it to Exchange for others to access and import it.

Comment (dukesphere) [reply modules/ROOT/pages/local-api-specification-tutorial.adoc:90]: @chris-campbell9 ☝️

Comment (dukesphere) [reply modules/ROOT/pages/local-api-specification-tutorial.adoc:90]: For desktop, it's truly local to your machine.

Terminology is confusing IMO. I think I tried to clarify it elsewhere in the docs.

File: modules/ROOT/pages/local-api-specification-tutorial.adoc (hunk 1 above)
Comment (dukesphere) [reply modules/ROOT/pages/local-api-specification-tutorial.adoc]: @chris-campbell9 Just that new style ☝️

File: modules/ROOT/pages/local-api-specification-tutorial.adoc (hunk 2 above)
Comment (dukesphere) [reply modules/ROOT/pages/local-api-specification-tutorial.adoc:90]: Oh, here's where I clarified: https://dev-docs-internal.kqa.msap.io/beta-code-builder/anypoint-code-builder/ping-locally-deployed-app

File: modules/ROOT/pages/local-api-specification-tutorial.adoc [hunk 3]
```diff
@@ -16,13 +18,16 @@ See xref:setup.adoc[] for more information.
 
 To create a new API specification:
 
-. In your web IDE instance, navigate to your Anypoint Code Builder view:
-+
-image::anypoint-code-builder-view.png[]
-. Select *Design an API*.
-+
-image::design-api-1.png[]
+. Open the Command Palette by pressing Ctrl+Shift+p (Mac) or Cmd+Shift+p (Windows),  and type _MuleSoft: Design an API_.
```
Comment (dmerlob) [inline modules/ROOT/pages/local-api-specification-tutorial.adoc]:
```suggestion
. Open the Command Palette by pressing Ctrl+Shift+p (Mac) or Cmd+Shift+p (Windows), and type _MuleSoft: Design an API_.
```

Comment (dmerlob) [reply modules/ROOT/pages/local-api-specification-tutorial.adoc]: If the build does not catch this, leave it as is.

**Review by dmerlob — APPROVED** [review]

**Pull request merged**